src/infrastructure/notification/notification.core-repository.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/infrastructure/prisma/prisma.service';
+import { NotificationRepository } from 'src/domain/notification/notification.repo';
+import { NotificationResponseEntity } from 'src/domain/notification/entity/notification.response.entity';
+import { NotificationsRequestDto } from 'src/interfaces/controller/notification/dto/notification.request.dto';
+
+@Injectable()
+export class NotificationCoreRepository implements NotificationRepository {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async findMany(args: NotificationsRequestDto) {
```

Comment:
```suggestion
  async findMany(args: NotificationsCommand) {
```
interfaces layer에 있는 객체가 infrastructure layer까지 오는건 의존생 위배가 되는 것 같습니다.
그래서 domain layer에서 xxxCommand 라는 dto 타입을 만들어서, 의존성이 위배되지 않게 한번더 converting 해주는 작업을 해주면 좋을 것 같습니다 :)

interfaces -> application -> domain <- infrastructure
이거 한번 더 생각해 보면 좋을 것 같아요!